### PR TITLE
Implement unarmed combat

### PR DESCRIPTION
**Pull request by ShadowCommander**
Adds a second loop for IAttack on the character if the item in hand doesn't have an IAttack component.

Currently it's using the same IAttack as MeleeWeaponComponent.

### CODE REVIEW

**Comment by AJCM-git** [comment]
fixes #395

File: Content.Server/GameObjects/Components/Weapon/Melee/UnarmedCombatComponent.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿
+using Content.Server.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.GameObjects.Components.Weapon.Melee
+{
+    [RegisterComponent]
+    public class UnarmedCombatComponent : MeleeWeaponComponent
```
Comment (Zumorica) [inline Content.Server/GameObjects/Components/Weapon/Melee/UnarmedCombatComponent.cs:8]: You created this class here, but it isn't really used elsewhere in your changes, right?

Comment (ShadowCommander) [reply Content.Server/GameObjects/Components/Weapon/Melee/UnarmedCombatComponent.cs:8]: Used in the Human.yml
I could just put the MeleeWeaponComponent on the human, but there should probably be some differences.

Comment (Zumorica) [reply Content.Server/GameObjects/Components/Weapon/Melee/UnarmedCombatComponent.cs:8]: Ah, I completely missed that this inherited MeleeWeaponComponent--

Comment (RemieRichards) [reply Content.Server/GameObjects/Components/Weapon/Melee/UnarmedCombatComponent.cs:8]: zummkopf

**Comment by RemieRichards** [comment]
he merge he own pr
aboos!!!

**Pull request merged**